draft-ietf-acme-acme.md

```diff
@@ -1178,16 +1180,17 @@ Content-Language: en
 
 ### External Account Binding
 
-The server MAY require a value to be present for the "external-account-binding"
-field.  This can be used to an ACME account with an existing account in a
-non-ACME system, such as a CA customer database.
+The server MAY require a value for the "external-account-binding" field to be
+present in "new-account" requests.  This can be used to an ACME account with an
+existing account in a non-ACME system, such as a CA customer database.
 
-To enable ACME account binding, a CA needs to provision the ACME client with a
-MAC key and a key identifier. The key identifier MUST be an ASCII string. The
-MAC key SHOULD be provided in base64url-encoded form, to maximize compatibility
-between provisioning systems and ACME clients.
+To enable ACME account binding, a CA needs to provide the ACME client with a
+MAC key and a key identifier, using some mechanism outside of ACME. The key
+identifier MUST be an ASCII string. The MAC key SHOULD be provided in
+base64url-encoded form, to maximize compatibility between provisioning systems
```

Comment:
nit: "provisioning systems" should maybe be clarified as "non-ACME provisioning systems"?